documentation/translation.md

```diff
@@ -4,21 +4,21 @@ The application is being designed and written in English and simultaneously bein
 
 ## Tools
 
-The common subproject contains a script (`src/tooling/checkIntl.js`) to help manage the react-intl translations files. It has been written to run without any dependencies or compilation. It is expected to be used along with the [formatjs cli](https://formatjs.io/docs/tooling/cli).
+The i18n subproject contains a script (`dist/cli.js`) to help manage the react-intl translations files. It has been written to run without any dependencies or compilation. It is expected to be used along with the [formatjs cli](https://formatjs.io/docs/tooling/cli).
 
 ### Bulk Translation
 The checkIntl script can be run with different flags and options. For more details on how individual options work, see the checkIntl file itself. In practice, it is easiest to save the commands, with options included, as **package.json** scripts.
 
-Note: each subproject using react-intl (e.g. common, apps/web, etc.) requires its own set of commands, and must be managed separately.
+Note: each subproject using react-intl (e.g. packages/i18n, apps/web, etc.) requires its own set of commands, and must be managed separately.
 
 For example, to ensure translations in the apps/web project are up to date:
 1. Run `npm run intl-extract` in the project you are managing (in this case, apps/web).
-2. Run `npm run check-intl-web` (from the /frontend/common folder). This generates a **untranslated.json** file in the apps/web project's lang folder.
+2. Run `npm run check-intl:web` (from the root folder). This generates a **untranslated.json** file in the apps/web project's lang folder.
```

Comment:
I don't know a lot about Turbo... would it be possible to set it up such that "npm run check-intl" would run both check-intl commands in parallel, like most other commands?

Reply:
When I was writing it, I couldn't figure it out but now that i revisit it, I may have something in mind 🤔

Reply:
I think I have gotten all the important ones [remove other references to frontend folder](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5697/commits/bcb1eecc028e395da66ad0f1e3579f9d5e2c63bd)

Reply:
Cool, well this one's not a blocker, anyway.